CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Fix: Move vertex only if its position changes
```

Comment:
Previous behaviour was already like this, so maybe "Only edit edge layers if the edge position changes" describes this fix better?

Reply:
Sounds good, I rephrased your suggestion a bit